### PR TITLE
users-can-create-an-entry - Henry Pham

### PR DESCRIPTION
**Pull request by hqpham98**
Kapture 2023-11-14 at 10 20 42

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,13 +16,65 @@
...
+        <div data-view="entry-form">
+          <form>
+            <div class="row">
+              <div class="column-full">
+                <h1>New Entry</h1>
+              </div>
+            </div>
+            <div class="row">
+              <div class="column-half">
+                <img src="images/placeholder-image-square.jpg" />
```
Comment (shawnkost) [inline index.html]: Missing an `alt` attribute

File: index.html [hunk 2]
```diff
@@ -16,13 +16,65 @@
...
+              </div>
+              <div class="column-half entry-column">
+                <div>
+                  <label for="title">Title</label>
+                </div>
+                <div>
+                  <input type="text" id="title" required />
+                </div>
```
Comment (shawnkost) [inline index.html]: You can wrap these inside of a `div`, but you can also just apply `display: block` to the elements to get them on a new line

**Comment by hqpham98** [comment]
image

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good! One change requested here, and one suggestion

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**